samples/nextjs/scripts/generate-config.ts

```diff
@@ -33,6 +34,7 @@ export function generateConfig(configOverrides?: { [key: string]: string }): voi
   // based on other config settings.
   const computedConfig: { [key: string]: string } = {};
   computedConfig.graphQLEndpoint = '`${config.sitecoreApiHost}${config.graphQLEndpointPath}`';
+  computedConfig.sitecoreServicesApiKey = 'config.sitecoreApiKey';
```

Comment:
So, we have `SITECORE_API_KEY` and `SITECORE_SERVICES_API_KEY` and here you are using `config.sitecoreApiKey` should we define separate `config.sitecoreServicesApiKey`? In this case it should be added to CLI. If I will add `sitecoreApiKey` (not services key) in this case it will not work as expected.

Reply:
This line results in the following line in `temp/config.js`:
`config.sitecoreServicesApiKey = process.env.SITECORE_SERVICES_API_KEY || config.sitecoreApiKey;`

So it works as expected: if a customer specifies the SITECORE_SERVICES_API_KEY environment variable, its value is used, otherwise, the config.sitecoreApiKey is used.